src/app/not-found/not-found.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="margin-spacing" [style.height.px]="winHeight">
+  <div class="txt-container">
+    <div class="title">this loan does not exist</div>
+    <div class="miscellany"></div>
+    <div class="subtitle">Oooops, something went wrong</div>
+  </div>
+  <div class="img-container"></div>
+  <div id="clouds">
```

Comment:
The clouds don't re-appear outside off the screen

Reply:
Done